components/authn-service/tokens/pg/adapter.go

```diff
@@ -72,29 +102,14 @@ func (a *adapter) ResetToV1(ctx context.Context) error {
 }
 
 func (a *adapter) insertToken(ctx context.Context,
-	id string, description string, value string, active bool, projects []string, isLegacyToken bool) (*tokens.Token, error) {
+	id string, description string, value string, active bool, projects []string) (*tokens.Token, error) {
 
 	t := tokens.Token{}
 	// ensure we do not pass null projects to db and break the not null constraint
 	if projects == nil {
 		projects = []string{}
 	}
 
-	// we do not wish to validate projects for the A1 legacy token case
-	// since that is an internal DB call kicked off as part of the server startup code
-	// with no subjects
-	if isLegacyToken {
```

Comment:
This is cool: The logic gate is reversed here: we want to do the following code when isLegacyToken is false. But both callers also passed inverted values: true for false, and false for true. Just goes to prove that, yes, two wrongs make a right. 😁 

(In any case, this is now handled automatically because the legacy token creator does not call `validateTokenInputs`.)